src/OpenTelemetry.AutoInstrumentation.Native/util.cpp

```diff
@@ -174,4 +174,16 @@ WSTRING TokenStr(const mdToken* token)
     return s;
 }
 
+WSTRING HResultStr(const HRESULT hr)
```

Comment:
I'm using this function just in the new logs added in this PR but will do a separate one to add to any other location logging HRESULT.